src/Components/RocketsContainer.js

```diff
@@ -0,0 +1,33 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getRocket } from '../redux/rockets/rockets';
+import Rocket from './Rocket';
+import '../css/RocketsContainer.css';
+
+const RocketsContainer = () => {
+  const rockets = useSelector((state) => state.rocket);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    if (!rockets.length) {
+      dispatch(getRocket());
+    }
+  // eslint-disable-next-line
+  }, []);
```

Comment:
- [ ] I strongly suggest you don't disable any linter rules, we use linters to ensure that we ship high-quality code following certain best practices and standards and maintaining a consistent structure! 🤓 I see that the problem here is that it asks for `rockets.length` and `dispatch` to be in the dependency array, adding them would not affect the functionality of your app since we don't expect these dependencies to change during the lifecycle of the app, so there's no risk for an unexpected re-render. Please consider fixing this! 🔧